manifests/a/ashisoft/dff/8.1.0.1/ashisoft.dff.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: ashisoft.dff

```

```suggestion
PackageIdentifier: Ashisoft.DuplicateFileFinder
```
Edit filename and foldername as well.